percona/controller/pgcluster/extensions.go

```diff
@@ -0,0 +1,101 @@
+package pgcluster
+
+import (
+	"fmt"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+
+	v2 "github.com/percona/percona-postgresql-operator/pkg/apis/pgv2.percona.com/v2"
+)
+
+func GetExtensionKey(pgMajor int, name, version string) string {
+	return fmt.Sprintf("%s-pg%d-%s", name, pgMajor, version)
+}
+
+func ExtensionRelocatorContainer(image string, imagePullPolicy corev1.PullPolicy, postgresVersion int) corev1.Container {
```

Comment:
I would comment this functions to explain it and give a bit of context, like relocates what from where and why, even more since the logic is inside `relocate-extension.sh`.